content/spinup.md

```diff
@@ -0,0 +1,133 @@
+# Setup CTSM
+
+For running CTSM on the Norwegian cluster you will be affilicated with a project account. 
+
+To check which project you are affiliated with can run by typing
+
+    [~/HOME]$ projects
+
+after logging to a remote HPC-computer. This will return something that looks like this
+
+    nn2806k
+    nn1000k
+
+In the example above, two projects can be used (nn2806k and nn1000k).
+Then make sure you choose the right project when running CTSM.
```

Comment:
Should we be more specific?
Which is the right project? 
In the example, nn2* is the "general" MetOS project. nn1* was the master student project on able - I don't know how this is handled now. 
> The right project is the one that your account accounts on (the one that you wrote into the form when you applied for computing time) -> `cost`